assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -265,6 +265,12 @@ function APIProvider({ children }) {
     [users]
   );
 
+  const getCurrentUser = useCallback(() => {
```

Comment:
This needs to be added to `assets/src/edit-story/karma/fixture/fixture.js` as well

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/6132/commits/7232b7ba488b62de1da87355c577d92632161adb

Reply:
@spacedmonkey Hmm I meant adding only this method to the existing API provider stub. Stubbing the whole CurrentUserProvider is not necessary 🤔

Reply:
Sorry, wrong end of the stick here. Fixed in https://github.com/google/web-stories-wp/pull/6132/commits/8cdb251c4736146bfa6a6799a5eb5fee5443e6c5